core/drivers/crypto/stm32/common.h

```diff
@@ -8,7 +8,42 @@
 
 #include <tee_api_types.h>
 
+enum stm32_cipher_ip_id {
+	CRYP_IP,
+	SAES_IP,
+};
+
+/*
+ * Macro to manage bit manipulation when we work on a local variable
+ * before writing only once to the hardware register.
+ */
+#define CLRBITS(v, bits)		((v) &= ~(bits))
```

Comment:
This macro seems a bit dangerous. What if `v` is a 64-bit value and `bits` is a 32-bit value?
Generally, I think that the quality of helper macros like these would be improved by making them available via `libutils`. They would receive more reviews. Naturally, it's quite likely that they'd have to be changed a bit to make everyone happy too.

Reply:
The register of the CRYP and SAES are only 32 bits so the mismatch 64 bits/32 bits is not a huge concern right now for us but nevertheless it's a good point.
If this kind of helper could be integrated in `libutils`, I need some rework before proposing it.

Reply:
`libutils` already provide macros to manage bit field so I rework stm32_saes to use `set_field_u32` and remove the macro in common.h. done in [`68f29e7`](https://github.com/OP-TEE/optee_os/pull/6168/commits/68f29e7fa616a35d8ac77c41ee781bd8d808ce52)